djstripe/stripe_objects.py

```diff
@@ -262,6 +284,54 @@ def _stripe_object_to_invoice(cls, target_cls, data):
         return target_cls._get_or_create_from_stripe_object(data, "invoice")[0]
 
     @classmethod
+    def _stripe_object_to_invoice_items(cls, target_cls, data, invoice):
+        """
+        Retrieves InvoiceItems for an invoice.
+
+        If the invoice item doesn't exist already then it is created.
+
+        If the invoice is an upcoming invoice that doesn't persist to the
+        database (i.e. ephermeral) then the invoice items are also not saved.
+
+        :param target_cls: The target class to instantiate per invoice item.
+        :type target_cls: ``StripeInvoiceItem``
+        :param data: The data dictionary received from the Stripe API.
+        :type data: dict
+        """
+        lines = data.get("lines")
```

Comment:
Nevermind, got it based on your other comments that it's between docstring and body. ;-)